### DISCUSSION TITLE
How to create users and roles?

### DISCUSSION THREAD

**Discussion by currynaa** [dbgate/dbgate#879] [Q&A]
Is it possible to have multiple users with different roles in DbGate? If it is, how do I create users and assign roles?

TIA

**Comment by janproch** [comment]
Yes, https://dbgate.org/docs/admin.html

Please use this docker image: https://hub.docker.com/r/dbgate/dbgate-premium

**Reply by currynaa** [reply]
is there any way to make it multiuser without premium? is it possible to make it happen just by configuring the .env

**Comment by janproch** [comment]
there are some limited ways, how to configure multiuser without premium - https://dbgate.org/docs/env-variables.html

but if you need bigger flexibility, comfort and support, you need premium for this

**Reply by Faig92** [reply]
@janproch could you kindly provide basic example of configuration with multiple users via oauth authentication.
For instance i needed hide connections list for different users.
